### ISSUE TITLE
Droptune logo

### ISSUE THREAD

**Issue by Shpigford** [Shpigford/droptune#8]
We need a logo. 😄

**Comment by Shpigford** [comment]
This is where I'm at, but it's had pretty mixed reception: https://twitter.com/Shpigford/status/1029042731010670592

<img width="850" alt="screen shot 2018-08-13 at 11 52 57 am" src="https://user-images.githubusercontent.com/35243/44045844-761c581a-9eef-11e8-8464-50489e7cc15e.png">

**Comment by jhnwlsn** [comment]
Love the idea of this app, Josh! Would love to help out here. I saw the twitter threads, but are there any other concepts/ideas that have been floating around in your head? What are you aiming to communicate? Influences?

**Comment by madebysan** [comment]
Hey Josh, I saw your tweet. 

I tried to incorporate the D, the play icon and fire (_drop it like it's hot_). The result is not mind-blowing but maybe is useful as a starting point. The typography used was Gilroy.

 Feel free to use or tweak as you wish.

[Download SVG/sketch](https://drive.google.com/open?id=1un8m1u7_PWwX9Wd9EVVEZrJ7KYw8S1c-)

![droptune](https://user-images.githubusercontent.com/1485035/44057544-81e2d6f4-9f19-11e8-801d-10e9f7bdff3f.png)
